scipy/stats/_qmc.pyi

```diff
@@ -0,0 +1,153 @@
+# Prevents annotations from being evaluated during runtime,
```

Comment:
@rgommers was this something that you reviewed? Shall we consider it done?

Reply:
Yes I reviewed all of this, it's good to go I believe. I'll resolve this comment.